### DISCUSSION TITLE
透明模式是否可以增加？

### DISCUSSION THREAD

**Discussion by Gala-TA** [galacean/discussion#91] [Editor] [answered]
透明模式是否可以有以下几类：
Blend SrcAlpha OneMinusSrcAlpha // 传统透明度
Blend One OneMinusSrcAlpha // 预乘透明度
Blend One One // 叠加
Cutoff //像素裁切

by：信州

**Comment by GuoLei1990** [comment]
你说的是编辑器中  PBRMaterial  暴露的透明模式么？ 

- Blend SrcAlpha OneMinusSrcAlpha // 传统透明度 【已有】
- Cutoff //像素裁切 【已有】

以下两者没有，内置的 Additive 模式为 Blend SrcAlpha One，有些区别！
- Blend One OneMinusSrcAlpha // 预乘透明度【没有】
- Blend One One // 叠加 【没有】

后面上线的 Shader  框架可以灵活的编辑任意混合模式的  blendMode 自定义  Shader，如果是想在 引擎标准的 PBRMaterial 中增加可能需要调研一下上面两种缺失混合模式的常用性

**Comment by GuoLei1990** [comment, marked as answer]
ShaderLab 已上线，混合状态配置详见文档：https://galacean.antgroup.com/engine/docs/latest/cn/graphics-shader-lab#%E5%85%A8%E5%B1%80%E5%8F%98%E9%87%8F
